lightwood/encoders/time_series/rnn.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import unicode_literals, print_function, division
+from lightwood.encoders.time_series.helpers.rnn_helpers import *
+
+import torch
+import torch.nn as nn
+from torch import optim
+
+import numpy as np
+
+class RnnEncoder:
+
+    def __init__(self, encoded_vector_size=4, train_iters=75000, stop_on_error=0.8, learning_rate=0.01, is_target=False):
```

Comment:
I'm not really sure it makes sense to have: `train_iters`, `stop_on_error` and `learning_rate` as part of the interface here, since we don't for the other encoders that require training.

It might make more sense to set them as public variables (without the `_`) and have them be overidden dynamically via the config as we do with the other encoder when we initialize the data source:

```
                encoder_attrs = config['encoder_attrs'] if 'encoder_attrs' in config else {}
                for attr in encoder_attrs:
                    if hasattr(encoder_instance, attr):
                        setattr(encoder_instance, attr, encoder_attrs[attr])
```

Reply:
they are public variables, and you can set them via params on config, as such its ok to have these params here so the encoder can be used standalone, to facilitate moving to a factory pattern, separate PR for that